### DISCUSSION TITLE
Build platform initial layer down speed

### DISCUSSION THREAD

**Discussion by tgob** [sn4k3/UVtools#666] [Q&A] [answered]
Is it possible to add a feature to reduce the build platform downward speed for initial layer movement?

The issue I'm having is the return speed on my Elegoo Saturn 2 is too fast.  It creates significant hydraulic forces that result in the initial layer being too thick (about 0.27 mm when it should be 0.05 mm.  The thick initial layer causes the next ~ 10 layers to be compressed.

If I set Z=0 to the build plate on the FEP which does sit on the LCD then, with resin in the tank, on starting a print the the Z motor actually skips steps during the final second of the initial move.

Increased rest time before exposure doesn't seem to reduce the problem.  I figure this is because the build platform is not getting to the first layer height.  

![Perfect](https://user-images.githubusercontent.com/20181668/221158656-6f996a9d-fdf1-4207-973d-961ddd86199c.jpg)

**Comment by sn4k3** [comment, marked as answer]
It's possible to edit settings for layer range under Tools - Edit print parameters. But that will not solve the problem...
You will need to wait some time for build plate to settle, and that strategy is already on UVtools under Suggestions, check [this](https://blog.honzamrazek.cz/2022/07/step-by-step-guide-on-perfect-bed-adhesion-and-elephant-foot-removal-in-uvtools-3/)

**Comment by tgob** [comment]
Thank you for your reply Tiago.  Much appreciated.

I have:

- explored all of the editable print parameters in UV Tools but there are no options for adjusting build plate return speed.  This isn’t the lift or retract speed.  It is the downward speed of the build plate.
- previously found your link.  Yes, it removes elephant’s foot but it doesn’t fix my consistently too thick bottom layer.  It was this site that lead me to your excellent software.
- Tried extensive initial layer pre-cure delays (up to 60 seconds) but the very first layer remains too thick (always about 0.27 mm).
- Leveled the build plate with no change in performance.
- Removed the axial play with thrust bearing at the top and bottom of the Z lead screw (see:  https://www.facebook.com/groups/elegoosaturn/posts/1290831881501256/ ). 
- Leveled the resin vat so the FEP is sitting directly on the LCD.
- Changed Z=0 from 0 to 0.1 to 0.2 mm with no effect.

 
I am convinced this an hydraulic pressure issue.  The LCD, build plate and likely the printer deck are deflecting.  Wait before cure doesn’t help because the Z motor skips steps with Z=0 at true 0.  The only way to remove this is to slow down the return speed of the buld plate or drill holes or mill channels in the build plate.

Is the return speed controllable in the .cbt file or is this a printer firmware setting?
 
Any other suggestions that you might offer would be welcome.

With kindest regards,
Tim

**Reply by sn4k3** [reply]
The return speed to the curing height should be the RetractSpeed.
Which version of CTB you run? To be able to use proper wait times you must use CTBv4.
Additionally, you need an empty first layer to bypass firmware stupid constrain where it never waits on first layer (UVtools create that when apply suggestion)

**Comment by tgob** [comment]
Thank you again for an informative and prompt reply Tiago.
I‘ll significantly reduce the bottom layer retract speed (using UV Tools of course) and report back.

With kindest regards,
Tim

**Reply by bryanseger** [reply]
If it's anything like a Mars 3 (and it likely is), that initial downward speed when the head homes itself to the Z=0 sensor is controlled by the firmware, not the speeds set in your print file. If you download the gcode firmware, it'll be in Chinese, but open it in Notepad ++, copy/paste it into Google Translate, then bring it back into a new note document. From there, you can find the speeds and reduce them. That'll help reduce the number of steps missed at the initial homing process. You'll also need to reduce the downward speed on the first few layers of the print so you don't lose steps there either.

**Comment by tgob** [comment]
Thanks for your email Bryan.

I haven’t considered changing the firmware yet.  Life would be much easier if this were open source and documented.  I cannot even find a document that describes .ctb file formats.

 But the Z motor isn’t skipping steps.  The problem is the axial play in the Z axis ball screw in combination with the initial layer height and relatively large area of the build plate.  Counter-intuitively reduced speed actually reduces resin flow due to the thixotropic properties of the resin.

You can read about my experiments to date at http://turtlesarehere.com/html/axial-play.html and http://turtlesarehere.com/html/first-layer-quest.html 

My prints are now consistently within 0.05 mm subject to the limitations of resin shrinkage.  Here are the measurements from a calibration print completed yesterday with 3D printed ball screw thrust bearing housings.

 With kindest regards,
Tim
